README.md

```diff
@@ -1,108 +1,75 @@
+| Model          |   de      |   es      |   fr      |   it      |   ja      |   ru      |   zh       |
+|----------------|-----------|-----------|-----------|-----------|-----------|-----------|------------|
+|LASER           |   92.70   |   88.75   |   90.80   |   85.93   |   85.15   |   84.65   |  88.98     |
+| MultiBERT      |   94.0    |   95.15   |   93.20   |   85.82   |   87.48   |   86.85   |  90.72     |
+| MultiFiT       | **95.90** | **96.07** | **94.77** | **90.25** | **90.03** |  **87.65**| **92.52**  |
 
-Directory structure after changes to the way we process wiki dumps.
-```
-data
-├── imdb
-│   ├── aclImdb
-│   ├── imdb_lm
-│   └── tmp
-├── wiki
-│   ├── de-100
-│   │   └── models
-│   ├── de-100-unk
-│   │   └── models
-│   ├── de-2
-│   │   └── models
-│   ├── de-2-unk
-│   │   └── models
-│   ├── de-all
-│   │   └── models
-│   ├── wikitext-103
-│   │   └── models
-│   └── wikitext-2
-│       └── models
-├── wiki_dumps
-├── wiki_extr
-│   └── de
-│       ├── AA
-│       ├── AB
-...
-        └── CC
-└── xnli
-    ├── XNLI-1.0
-    └── XNLI-MT-1.0
-        ├── multinli
-        └── xnli
-```
-
-## how to contribute
-We have a fork of fastai to propose changes to fastai.text, with a branch for this project:
- https://github.com/n-waves/fastai/tree/ulmfit_multilingual  
-
-Let us know that you want to start collaboration on fastai forum thread: [Multilingual ULMFIT](https://forums.fast.ai/t/multilingual-ulmfit/28117)
-and you will get access to both repositories.
+### Amazon CLS
+Sentiment classification results on CLS dataset [Prettenhofer and Stein, 2010](https://dl.acm.org/citation.cfm?doid=2036264.2036277)
 
-- Follow the [developer installation of fastai](https://github.com/fastai/fastai#developer-install)
-- Add n-waves/fastai as additional remote as described here: https://help.github.com/articles/adding-a-remote/
+|          |         DE            |          FR           |               JA     |
+|----------|-----------------------|-----------------------|----------------------|
+| MultiBERT| 86.05 / 84.90 / 82.00 | 86.15 / 86.90 / 86.65 | 80.87 / 82.83 / 79.95|
+| MultiFiT | 93.19 / 90.54 / 93.00 | 91.25 / 89.55 / 93.40 | 86.29 / 85.75 / 86.59|
 
-Here is what I did:
-```bash
-$ cd fastai
-$ git remote add n-waves https://github.com/n-waves/fastai.git
-$ git remote -v 
-n-waves	https://github.com/n-waves/fastai.git (fetch)
-n-waves	https://github.com/n-waves/fastai.git (push)
-origin	https://github.com/fastai/fastai.git (fetch)
-origin	https://github.com/fastai/fastai.git (push)
 
-$ git fetch n-waves
-$ git checkout ulmfit_multilingual
-Branch 'ulmfit_multilingual' set up to track remote branch 'ulmfit_multilingual' from 'n-waves'.
-Switched to a new branch 'ulmfit_multilingual'
-
-$ git push --set-upstream n-waves ulmfit_multilingual  # to automatically push ulmfit_multilingual branch to the n-waves repo
+## How to use it with fastai v1.0
+You can use the pretrained models with fastai library as follows:
+```
```

Comment:
This code snippet is great to make things understandable. For completeness, it would be good to add the relevant imports:
```
from fastai.text import *
import ulmfit
```

Reply:
As I look at the imports, the ulmfit name might be confusing given the multift repository and paper. I will change that to `multifit`:
```
from fastai.text import *
import multifit
```